src/pages/ErrorPage.js

```diff
@@ -1,7 +1,9 @@
 import React from "react";
 
+import ErrorComponent from "../components/Errors/ErrorComponent";
+
 const ErrorPage = () => (
-    <p>Something is wrong in the backstage. :(</p>
+    <ErrorComponent Message={"Something is wrong in the backstage. :("}/>
```

Comment:
The old message was just a placeholder. Please change to something more formal like "Something unexpected prevented us from fulfilling your request. Please try again later, and if the problem persists, contact us at ni@aefeup.pt (make this an actual `mailto` link)"